src/main/resources/application.conf

```diff
@@ -293,7 +293,7 @@ mantis {
     target-block-offset = 500
 
     # How often to query peers for new blocks after the top of the chain has been reached
-    check-for-new-block-interval = 1.seconds
+    check-for-new-block-interval = 10.seconds
```

Comment:
Why this ten fold increase? (i don't tell it is bad just want some justification)

Reply:
I should probably add a comment on this to the code, but I changed to reflect the [value set in Geth](https://github.com/ethereumproject/go-ethereum/blob/master/eth/sync.go#L137). This change also makes sense as we are processing new blocks in two different ways (handling NewBlock messages and by requesting them with `check-for-new-block-interval` frequency) instead of only requesting them.